NDB.Covid19/NDB.Covid19/Locales/ur.json

```diff
@@ -50,7 +50,7 @@
   "CONSENT_SIX_PARAGRAPH": "میں Smittestopp چلانے اور استعمال کرنے پر رضامند ہوں۔ اسکا مطلب ہے کہ میرا فون کانٹیکٹ keys بنا کر ان دوسرے فونز کے ساتھ انکا تبادلہ کرے گا جو میرے آس پاس ہوں اور مجھے اطلاعات مل سکیں گی کہ میرے قریب کوئی ایسا شخص رہا ہے جسے COVID-19 انفیکشن ثابت ہوا ہے۔ نہ مجھے اور نہ کسی اور کو پتہ چلے گا کہ انفیکشن کسے ہے۔",
   "CONSENT_SEVEN_LINK_TEXT": "پرائیویسی پالیسی کھولیں",
   "CONSENT_SEVEN_LINK_URL": "https://www.fhi.no/en/about/smittestopp/use-of-smittestopp-privacy-policy/",
-  "CONSENT_GIVE_CONSENT": "میری طرف سے اجازت ہے",
+  "CONSENT_GIVE_CONSENT": "میری طرف سے\n اجازت ہے",
```

Comment:
Just to double check, are you sure breakline is correctly placed?

Reply:
Unfortunately I do not speak Urdu, but I tried to place it in a place with a space, in the middle of the sentence, but perhaps it should be verified @mmn-nc